ResuMe/urls.py

```diff
@@ -14,15 +14,16 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
-from posts import views
+from django.urls import path, include
 from django.conf import settings
 from django.conf.urls.static import static
 
 
 urlpatterns = [
     path('admin/', admin.site.urls),
-    path('', views.posts, name='posts')
+    path('', include('posts.urls')),
+    path('users/', include('users.urls')),
+    path('direct/', include('direct_message.urls'))
```

Comment:
Do we want to add `/users/` for every user URL? Same with `/direct`

Reply:
I think that the URLs are well organized that way.
So if I want to search for a user's profile, I'll find it under `/users/`.
Regarding `direct`, I think it is convenient as well, and I saw that they use a similar URL on Instagram.
![image](https://user-images.githubusercontent.com/96747472/162623699-967c5826-e671-4e16-86a2-4b9e2e54e385.png)